exercises/flower-field/metadata.toml

```diff
@@ -0,0 +1,4 @@
+title = "Flower Field"
+blurb = "Help avoid stepping on the flowers in a flower garden."
```

Comment:
There is a single reference in this commit to a cutting room floor in the intro file, a single reference to a garden in the intro, and a single reference stepping on flowers in the metadata. The rest of the docs don't mention cutting rooms, floors or stepping on things. There's mention of counting flowers in fields and boards, though.

Do we want to make things more consistent about the location and goal?

Reply:
The web reference to the cutting room floor website is not in the intro file since I switched it to using wikipedia as the auto-linter was getting an http response code of forbidden 403.

Reply:
Adjusted blurb